hatchways/team-bruschetta
# feature-add-ui-profile-photo-in-profilesettings-#24

### What this PR does (required):
UI for uploading user's profile picture.

### Screenshots / Videos (front-end only):
![Screen Shot 2021-10-16 at 7 02 06 PM](https://user-images.githubusercontent.com/42471478/137606240-d161cca2-2096-4a64-a069-02cd4c25bdd5.png)
![Screen Shot 2021-10-16 at 7 31 12 PM](https://user-images.githubusercontent.com/42471478/137606745-a0b8eea4-f546-4d29-a3ac-e2c925b99e72.png)

### Any information needed to test this feature (required):

1. run the react app locally
2. login with credentials 
3. goto `localhost:3000/settings` you can see the UI

### Any issues with the current functionality (optional):

1. There is a bug fix related to routing. Please check that as well, I might be wrong (Login.tsx & useAuthContext.tsx)
2. As I mentioned during the standup I've accidentally started building the NavBart and the SideDraw components (1st screenshot). I'll be using the same code in a future ticket. for now I'm committing & deleting those components for future reference. So, only what you see in the 2nd screenshot will be pushed with this PR
